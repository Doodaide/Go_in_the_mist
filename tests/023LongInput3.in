place R19
place R18
place S19
place R17
place P19
place R16
place Q19
place R15
place P18
place Q15
place Q18
place Q16
place P17
place Q17
place O19