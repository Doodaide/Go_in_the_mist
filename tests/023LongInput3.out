Black wins!
R19R18S19R17P19R16Q19R15P18Q15Q18Q16P17Q17O19
Thank you for playing!
